### PR TITLE
Feature: v2 local deploy

### PR DESCRIPTION
**Pull request by shark0der**

### CODE REVIEW

File: contracts/modules/capital/MCR.sol [hunk 1]
```diff
@@ -229,8 +234,8 @@ contract MCR is IMCR, MasterAware {
    * @param code parameter code
    * @param val new value
    */
-  function updateUintParameters(bytes8 code, uint val) public {
-    require(master.checkIsAuthToGoverned(msg.sender));
```
Comment (roxdanila) [inline contracts/modules/capital/MCR.sol:233]: is this function used anywhere else? prolly worth getting rid of it

File: contracts/modules/token/TokenController.sol [hunk 2]
```diff
@@ -36,6 +36,10 @@ contract TokenController is ITokenController, LockHandler, LegacyMasterAware {
     claimsReward = claimsRewardAddress;
   }
 
+  function unlistClaimsReward() external {
+    token.removeFromWhiteList(claimsReward);
```
Comment (roxdanila) [inline contracts/modules/token/TokenController.sol:40]: added an issue to manage this in master instead

File: contracts/modules/capital/MCR.sol (hunk 1 above)
Comment (roxdanila) [reply contracts/modules/capital/MCR.sol:233]: added an issue for this, 1 more place where it needs to be replaced

**Review by roxdanila — APPROVED** [review]

File: contracts/modules/capital/MCR.sol [hunk 3]
```diff
@@ -73,12 +72,18 @@ contract MCR is IMCR, MasterAware {
...
 
+    // copy over values
     mcrFloor = previousMCR.mcrFloor();
-
     mcr = previousMCR.mcr();
     desiredMCR = previousMCR.desiredMCR();
     lastUpdateTime = previousMCR.lastUpdateTime();
+
+    // copy over parameters
+    mcrFloorIncrementThreshold = previousMCR.mcrFloorIncrementThreshold();
```
Comment (danoctavian) [inline contracts/modules/capital/MCR.sol:82]: we should add to the v2 fork test to validate all values are moved correctly.

We will be upgrading MCR.sol to get the new logic for the MCR gearing active cover amount computation.

File: contracts/modules/token/TokenController.sol (hunk 2 above)
Comment (danoctavian) [reply contracts/modules/token/TokenController.sol:40]: any particular reason to do this in master and not here?

**Review by danoctavian — APPROVED** [review]
LGTM

File: contracts/modules/token/TokenController.sol (hunk 2 above)
Comment (roxdanila) [reply contracts/modules/token/TokenController.sol:40]: I guess there are multiple contracts that require this

**Pull request merged**